packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -16,44 +16,106 @@ import {
 	__experimentalBlockSettingsMenuFirstItem,
 } from '@wordpress/block-editor';
 import { Popover } from '@wordpress/components';
+import { useState, useEffect, createPortal } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import BlockInspectorButton from './block-inspector-button';
 import { useResizeCanvas } from '../resize-canvas';
+import { __ } from '@wordpress/i18n';
 
-function VisualEditor() {
-	const inlineStyles = useResizeCanvas();
+export const IFrame = ( { children, head, styles, ...props } ) => {
+	const [ contentRef, setContentRef ] = useState();
+	const doc = contentRef && contentRef.contentWindow.document;
+
+	useEffect( () => {
+		if ( doc ) {
+			doc.body.className = 'editor-styles-wrapper';
+			doc.body.style.margin = '0px';
```

Comment:
Units are not needed for zero-values.
```suggestion
			doc.body.style.margin = '0';
```